verification/src/transaction_verifier.rs

```diff
@@ -640,7 +702,7 @@ impl<'a, DL: HeaderProvider> SinceVerifier<'a, DL> {
         if since.is_absolute() {
             match since.extract_metric() {
                 Some(SinceMetric::BlockNumber(block_number)) => {
-                    if self.block_number < block_number {
+                    if self.tx_phase.earliest_committed_block(self.block_number) < block_number {
```

Comment:
Pay attention to this. You change the `block_number` from `tip_number + 1` to `tip_number` [outside](https://github.com/nervosnetwork/ckb/pull/2652/files#diff-f17291b75b5bafb128afde15b639dea00e71c718713a389060fe31cd9e53a914R103).

As a result, after the above change, block_number is 1 less than the old code.
I think here it should use `<=`.

Reply:
I think the integration test  `ValidSince` could prove that current comparison is correct.